index.html

```diff
@@ -1,160 +1,163 @@
+        <span class="fa fa-bars hamburger" aria-hidden="true"></span>
+    </a>
+
+    <div>
+            <a href="/"><img class="logo" src="images/systers-logo.png" 
+                          alt="Systers - Anitab.org logo"></a>
+    </div>
+  
+  <nav id="main-menu" 
+        class="main-menu"
+        role="navigation"
+        aria-expanded="false"
+        aria-label="Main menu">
+    <a href="#main-menu-toggle" 
+        class="menu-close"
+        role="button"
+        id="main-menu-close"
+        aria-expanded="false"
+        aria-controls="main-menu"
+        aria-label="Close main menu">
+        <span class="sr-only">Close main menu</span>
+        <span class="fa fa-close" aria-hidden="true"></span>
+    </a>
+    <ul>
+        <li><a href="#join-systers">Join Systers</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#programs">Programs</a></li>
+        <li class="toggle"><a href="#main-menu">Contribute<span class="fas fa-chevron-down"></span></a>
+            <ul>
+              <li><a href="/newcomers">Newcomers</a></li>
+              <li><a href="/open-source-workflow">Open Source Workflow</a></li>
+              <li><a href="/member-levels">Member Levels</a></li>
+            </ul>
+        </li>
+         <li><a href="/contactus">Contact Us</a></li>
+        <li class="toggle"><a href="#main-menu">Others<span class="fas fa-chevron-down"></span></a>
+		    <ul>
+		      <li><a href="/code-of-conduct">Code of Conduct</a></li>
+              <li><a href="/reporting-guidelines">Reporting Guidelines</a></li>
+            </ul>
+		</li>
+    </ul>
+    </nav>
+	</div>
+    <a href="#main-menu-toggle" 
+    class="backdrop" 
+    tabindex="-1"
+    aria-hidden="true" 
+    hidden></a>
+  </header>
+
+    <div class="header"><h1>Welcome to Systers' Open Source!</h1></div>
+        <p>Systers is an international <a href="https://anitab.org/" target="_blank">Anita B.org</a> community encouraging the participation of all women involved in technology.</p>    
+    
+        <p>We are committed to providing a friendly, safe, and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, work experience, or religion (or lack thereof).</p>
+
+    <div class="subheader" id="join-systers"><h2>Join Systers</h2></div>
+        <p> In order to engage with the community, you can sign up on <a href="http://systers.io/slack-systers-opensource/" target="_blank">Systers Open Source Slack</a> and introduce yourself in the #intro channel, then subscribe to our <a href="http://systers.org/mailman/listinfo/systers-opensource" target="_blank">systers-dev mailing list</a> for more opportunities.</p>
+
+        <p>You can also attend the Systers Weekly Update meetings every Thursday at 12:00 Eastern Standard Time (17:00 UTC) on Zoom.  You'll find the meeting details on our <a href="/contactus.html">contact page</a>.</p>
+

```

Comment:
@Vertika-Shukla Looks like you missed removing `.html` from here.

Reply:
I think in @Vertika-Shukla's issue she has mentioned on fixing on the main navigation only. We can create a separate issue for this issue too. Anyways, @Vertika-Shukla are you fixing the line @sara-02 has mentioned here?